gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,417 @@
+id: gigamon
+metric_id: gigamon
+backend_only: false
+facets:
+  - facetType: list
+    groups:
+      - gigamon
+    name: App Id
+    path: gigamon.app_id
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: Application Name
+    path: gigamon.app_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Destination IP
+    path: gigamon.dst_ip
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Destination Packets
+    path: gigamon.dst_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: Destination Port
+    path: gigamon.dst_port
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: DNS Flags
+    path: gigamon.dns.flags.rcode
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: DNS ID
+    path: gigamon.dns.id
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: HTTP  User agent OS Family
+    path: gigamon.http.useragent_details.os.family.browser.family
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Http Code
+    path: gigamon.http_code
```

Comment:
Wait sorry, why did you change these to be under the "Gigamon" group and namespace? The only changes to add the `gigamon.` prefix to the path were the facets that were under the "Gigamon" group. These standard facets you had before were supposed to be unchanged. 

I found [this](https://github.com/Mrudula-Oruganti-Gigamon/integrations-extras/blob/afd3f7aecd8fcba6df9500b723bdaa537c898745/gigamon/assets/logs/gigamon.yaml) version of the file before you made the changes. You had all the other facets correctly handled under the respective facet groups like `Web Access`, and `DNS`. The only facets that needed appending a `gigamon.` prefix to the path were `app_name`, `dns_host`, and `dns_name`. All the other facets were meant to remain unchanged

Reply:
Thanks @vishalshah-dd  for the clarification. I will do respective changes. I will keep the standard attributes as such and will add the gigamon. prefix for the custom attributes.

Reply:
Hi @Mrudula-Oruganti-Gigamon could you please revert your last commit? Once that is done, please let me know and I will advise on next steps. 

As Vishal said, it looks like there were a lot of changes that weren't required or requested that have been added to the yaml files. Please revert and I will re-comment on what needs to be changed.

Reply:
@emarsha94 , I have made changes as per Vishal's suggestion in the latest commit and have added few more attributes and facets as per his statements. 
If i revert back those commit, i may need to perform all the other changes like timestamp, adding new facets again. Do you still want me to revert to the one which i did 3months back?
Is there any issue with the latest build ? Please help.

Reply:
Hi Mrudula, it looks like there were a lot of changes that were not asked and additional facets added that don't belong in the groups they were added to. At this point, I believe the next best step is to revert the most recent commit then follow Vishals comments **very carefully**: https://github.com/DataDog/integrations-extras/pull/2356#pullrequestreview-2270074906 

> -You've changed all existing facets to be under "Gigamon" and the gigamon. path prefix. This was only supposed to be done to the facets you originally had under "Gigamon".
- I see a bunch of new facets added that weren't there before, some of these should not be under "Gigamon" group and are standard facets (one example: gigamon.src_port this should be mapped to network.client.port. You can see a list of standard facets for logs [here](https://docs.datadoghq.com/standard-attributes/?product=log). Any facet that you have that has the same behavior as these standard facets should reuse the same paths mentioned in the linked doc, and should not be under the "Gigamon" facet group
- Need to remove the timestamp facet, and keep the value as just ts and then have a date remapper on ts. This should not be saved as gigamon.ts since it's a standard attribute in Datadog

Please read the above and review the linked documents and update as the above instructs. Examples were provided in this [comment](https://github.com/DataDog/integrations-extras/pull/2356#discussion_r1707218560) on exactly how the formatting should look.

Reply:
[gigamon.yaml.txt](https://github.com/user-attachments/files/16853308/gigamon.yaml.txt)


@emarsha94 , That's true. This is what we have done in our latest commit which we did 4days ago. Do we need to do "submit for review" everytime we commit from github? Attaching the updated yaml file (in .txt format) for reference.

Reply:
@emarsha94 , the comments that were given by @vishalshah-dd and you have been addressed . Please refer to the comments at the below link :

https://github.com/DataDog/integrations-extras/pull/2356#issuecomment-2320464187

This should be part of the build:
https://github.com/DataDog/integrations-extras/pull/2356/commits/58b166c2a9155f5e448e8f00c2f3e212d052cca2

please let me know if there is anything else.